examples/testerina-before-and-after-suite/testerina_before_and_after_suite.out

```diff
@@ -1,20 +1,17 @@
-# To run this sample, navigate to the directory that contains the
-# `.bal` file, and execute the `ballerina test` command.
-# You need to have the ballerina-tools distribution
-# installed in order to run this command.
-$ ballerina test testerina_before_and_after_suite.bal
-Compiling tests
-    testerina_before_and_after_suite.bal
+# For the moment `ballerina test` works only in projects.
+$ ballerina test sample_service
+Compiling source
+        dulvin/sample_service:0.1.0
```

Comment:
Shall we refrain from using personal info in the examples? Maybe use `foo` as the org name